### ISSUE TITLE
Example Client & Server

### ISSUE THREAD

**Issue by webfolderio** [trensetim/quic#1]
Hi,

Do you have plan to add a simple client & server example?

**Comment by thysce** [comment]
Yes. Once a MVP is implemented, I will host a sample server and provide example code. 

Currently the receiving and parsing of QUIC and TLS messages is implemented and the library can successfully decode the InitialPacket from both the example in the QUIC-Specification and the implementation of kwik (which only works as a client). 

Next step for me is to make a roadmap and implement assembling packets and datagrams to be sent. Third I will add a Handler for incoming messages. Once that is complete, that minimum viable implementation should be able to establish a connection. Thus than I will add a sample server.

**Comment by trensetim** [comment]
Yes. Once a MVP is implemented, I will host a sample server and provide example code.

Currently the receiving and parsing of QUIC and TLS messages is implemented and the library can successfully decode the InitialPacket from both the example in the QUIC-Specification and the implementation of kwik (which only works as a client).

Next step for me is to make a roadmap and implement assembling packets and datagrams to be sent. Third I will add a Handler for incoming messages. Once that is complete, that minimum viable implementation should be able to establish a connection. Thus than I will add a sample server.

**Comment by trensetim** [comment]
Do you plan to use QUIC in your Java application or do you want to support the development of this library?

**Comment by webfolderio** [comment]
Yes, I would like to use this library but I am not familiar with low level protocols like QUIC. I don't know how to support the development? I am open to suggestions.

**Comment by trensetim** [comment]
I could use some more developers to progress more ~quickly~ quicly. Maybe some will get involved if this project would be a little more popular. To get new devs a bit of orientation, I will provide some sort of roadmap and tickets to solve soon.

**Comment by trensetim** [comment]
Please note that as of now, the library is not production ready. I will let you know once that is reached
